src/OpenTelemetry/README.md

```diff
@@ -308,6 +306,19 @@ Core, Console.
 }
 ```
 
+Tip: You can find the "current working directory" by calling
```

Comment:
This would requiring make code change just to get "current working directory"? Perhaps an easier way for troubleshooting is find the process in [Process Explorer](https://docs.microsoft.com/en-us/sysinternals/downloads/process-explorer) -> double click on the process to pop up properties dialog -> "Image" tab -> "Current directory".

Reply:
Thanks for the tip. I've included it.